public/app/common/helpers/device/device-helpers.js

```diff
@@ -33,6 +34,13 @@ var Helpers = {
     return component.properties.capabilities[0];
   },
 
+  getDeviceUrl: function getDeviceUrl (device) {
+    var link = _.find(device.links, function getLinkToSelf (linkObject) {
+      return linkObject.rel[0] === 'self';
+    });
+    return link.href;
+  },
+
```

Comment:
Would avoiding a variable and thus writing `_.find(...).href` be less readable here?